src/libraries/Common/tests/AndroidTestRunner/AndroidTestRunner.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetFrameworkCurrent)</TargetFrameworks>
+  </PropertyGroup>
+  <!-- Since this is referenced by test projects that are restored for NetFrameworkCurrent, we need to suppor it for restored
+  at runtime this is a no-op as tests can't run in NetFrameworkCurrent on Android so we will never build this for that tfm. -->
```

Comment:
Why can't you condition the P2P for non .NET Framework?

Reply:
Didn’t try that. If that works I’ll put a follow up PR.